### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by MartinThoma**
See #27

### CODE REVIEW

**Comment by MartinThoma** [comment]
Oops, wrong one. #199 was it.

The Sphinx documentation is only a draft. If you like it, I can adjust it (e.g. add `converters`, `cursors`, `cursors`, `util`) and probably move the examples from the README to the Sphinx docs.

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,4 @@ __pycache__
 /.tox
 /build
 /pymysql/tests/databases.json
+docs/build
```
Comment (methane) [inline .gitignore]: EOL is required

Comment (MartinThoma) [inline .gitignore]: Done.

**Comment by methane** [comment]
Could you rebase?

**Comment by methane** [comment]
http://pymysql.readthedocs.io/en/latest/modules/connections.html
autodoc seems failed.

File: docs/source/modules/connections.rst [hunk 2]
```diff
@@ -0,0 +1,10 @@
+:mod:`pymysql.connections`
+==========================
+
+.. automodule:: pymysql.connections
+
+.. autoclass:: MysqlPacket
+.. autoclass:: FieldDescriptorPacket
```
Comment (methane) [inline docs/source/modules/connections.rst]: Packet classes are too lower level to documenting on read the docs.

File: docs/source/modules/connections.rst [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+.. autoclass:: Connection
+.. autoclass:: MySQLResult
+.. autoclass:: LoadLocalFile
```
Comment (methane) [inline docs/source/modules/connections.rst]: MySQLResult and LoadLocalFile are too low layer too.

**Pull request closed without merging**